index.html

```diff
@@ -0,0 +1,286 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Development course</title>
+</head>
+
+<body>
+  <div class="mobile-logo-wrap">
+    <div>
+      <a href="index.html">
+        <img src="Images/main-logo.png" alt="developer week global">
+      </a>
+    </div>
+  </div>
+
+  <div class="mobile-menu-wrap">
+    <div class="mobile-menu-button">
+      <a href="#" id="mobile-menu-button">
+        <img src="Images/mobile_menu.png" alt="Mobile menu button">
+      </a>
+    </div>
+
+    <div class="mobile-menu-display display-none" id="mobile-menu-display">
+      <div class="mobile-menu-item align-self-fx-end">
+        <button id="close-mobile-menu">&#10799;</button>
+      </div>
+
+      <div class="mobile-menu-item">
+        <a href="#">
+          <p>News</p>
+        </a>
+      </div>
+
+      <div class="mobile-menu-item">
+        <a href="#">
```

Comment:
- [ ] I think it will be better if you wrap all your navigations and logo in an `<header>` element as the `<header>` element represents a container for introductory content or a set of navigational links 👍🏽 so its important to add it to make your website semantic.